### ISSUE TITLE
Need a beans.xml to start

### ISSUE THREAD

**Issue by yblazart** [hammock-project/hammock#123]
Hello, I'm testing hammock solution.

But I'm facing multiple issues.
First of them is that, even if starting tutorial say that we only need to add a JaxRS endpoint, CDI is not started (with dist-microprofile).

I have to add an empty beans.xml.

Regards.

**Comment by derekm** [comment]
@yblazart -- there was an issue with the `rest-cxf` module in Hammock 2.0 which is fixed in Hammock 2.0-SNAPSHOT. `dist-microprofile` includes `rest-cxf`.

I'm using `rest-jersey` in the style of the hammock-diy example, and it works well in Hammock 2.0.

**Comment by yblazart** [comment]
2.1-SNAPSHOT you mean ? Yes I just tested.

sometimes work, sometime not, really strange. It's working when I do not see any line about atmosphere in logs. Doing a rebuild in idea then run...  rarely fail, but happen

**Comment by johnament** [comment]
Yes, basically sometimes CXF initializes and other times it doesn't.  There's two fixes that went in, one within CXF and one on my side.  I'll be upgrading to CXF 3.2.2 which should fix this completely.
